lua/nvim-tree/actions/moves/git.lua

```diff
@@ -0,0 +1,62 @@
+local explorer_node = require "nvim-tree.explorer.node"
+local lib = require "nvim-tree.lib"
+local moves_item = require "nvim-tree.actions.moves.item"
+
+local M = {}
+local MAX_DEPTH = 100
```

Comment:
Sensible, thank you.

Reply:
This is a limit I chose arbitrarily. If you see another value that would make more sense, lmk.

Reply:
That's fine. We can add an option like `actions.expand_all.max_folder_discovery` only if we need it.